cosmo_tester/test_suites/broker_management/conftest.py

```diff
@@ -0,0 +1,112 @@
+import os
+import time
+
+import pytest
+
+from cosmo_tester.framework.test_hosts import BootstrapBasedCloudifyManagers
+from cosmo_tester.framework import util
+
+
+@pytest.fixture()
+def brokers(cfy, ssh_key, module_tmpdir, attributes, logger):
+    for _brokers in _get_brokers(cfy, ssh_key, module_tmpdir,
+                                 attributes, logger):
+        yield _brokers
+
+
+@pytest.fixture()
+def broker(cfy, ssh_key, module_tmpdir, attributes, logger):
+    for _brokers in _get_brokers(cfy, ssh_key, module_tmpdir, attributes,
+                                 logger, instance_count=1):
+        yield _brokers[0]
+
+
+class JustHosts(BootstrapBasedCloudifyManagers):
+    def _bootstrap_managers(self):
+        pass
+
+
+def skip(*args, **kwargs):
+    return True
+
+
+def _get_brokers(cfy, ssh_key, module_tmpdir, attributes, logger,
+                 instance_count=3):
+    hosts = JustHosts(
+        cfy, ssh_key, module_tmpdir, attributes, logger,
+        number_of_instances=instance_count,
+    )
+
+    tempdir = hosts._tmpdir
+
+    cert_paths = {
+        'ca': {
+            'base': os.path.join(tempdir, 'ca.'),
+        },
+    }
+    for node in range(instance_count):
+        node = 'node' + str(node + 1)
+        cert_paths[node] = {
+            'base': os.path.join(tempdir, node + '.'),
+        }
+    for paths in cert_paths.values():
+        paths['cert'] = paths['base'] + 'crt'
+        paths['key'] = paths['base'] + 'key'
+    util.generate_ca_cert(cert_paths['ca']['cert'], cert_paths['ca']['key'])
+
+    try:
+        for node in hosts.instances:
+            node.verify_services_are_running = skip
```

Comment:
a better way would be to make the TestHosts subclass we use, use a VM class that doesnt implements all that shit, ie. probably the base `VM`. I don't currently see a good way to do that but maybe you will